extensions/libarchive/FocusArchiveEntry.h

```diff
@@ -64,7 +64,7 @@ class FocusArchiveEntry : public core::Processor {
    public:
     explicit ReadCallback(core::Processor*, utils::file::FileManager *file_man, ArchiveMetadata *archiveMetadata);
     int64_t operator()(const std::shared_ptr<io::InputStream>& stream) const;
-    bool isRunning() {return proc_->isRunning();}
+    bool isRunning() const {return proc_->isRunning();}
```

Comment:
should this be marked `override`, as well?

Reply:
This method was unused so I removed it in a20c3a130d80bd73e4383b8161e23c52c77fb1fd